lib/stores/FileStore.ts

```diff
@@ -17,11 +17,13 @@ type Store = {
   get(key: string): File | undefined
   set(key: string, value: File): void
   delete(key: string): void
+  all: Record<string, File>
```

Comment:
I think this should not return all records since this would require that all records are read to memory. Maybe `all: string[]` would be more suitable? This would return only the keys.

Reply:
That's true.

 It is good to have common interface for all possible config stores (could be file, redis or memory based, or really anything else) and for that reason it is not ideal to design such interface based on specific implementation that has very poor performance.

If we go with more general interface for cofig stores then `configstore` could use an adapter to make it compatible.

Reply:
Yes. Can do/talk about that in another PR.